enterprise/api/v1/serializers.py

```diff
@@ -312,6 +319,8 @@ def validate_username(self, value):
         try:
             self.user = User.objects.get(username=value)
         except User.DoesNotExist:
+            error_message = '[Enterprise API] User with username {} does not exist.'.format(value)
```

Comment:
we should be more specific here as well; what operation failed due to this error?

Reply:
Done